test/e2e/benchmark/manifest.go

```diff
@@ -80,6 +80,8 @@ type Manifest struct {
 	GovMaxSquareSize int64
 
 	DisableBBR bool
+
+	GenesisAppVersion uint64
```

Comment:
What's this for?

Reply:
looks like just to set the app version in the genesis

cc @staheri14